Sources/FirestoreManager/FirestoreReference.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FirestoreReference.swift
+//  
+//
+//  Created by Anna Münster on 03.06.22.
+//
+
+import Firebase
+
+public protocol ReferenceProtocol {
+    var rawValue: String { get }
+    var parent: ReferenceProtocol? { get }
+}
+
+extension ReferenceProtocol {
+    func reference(parentId: String? = nil) -> CollectionReference {
+        if let parent = parent, let parentId = parentId {
```

Comment:
I guess the parentId here is to prepare for nested fetching/creating, right? Maybe we can add a comment for now to make that clear.